benchmark/benchmark.php

```diff
@@ -12,7 +12,8 @@
 function main() {
     global $storeResult;
 
-    $data = [];
+    $branch = runCommand(['git', 'rev-parse', '--abbrev-ref', 'HEAD'], dirname(__DIR__));
+    $data = ['branch' => $branch];
```

Comment:
is this change enough for https://github.com/php/php-src/blob/master/benchmark/generate_diff.php to show diff vs. the X.Y version HEAD?

Reply:
I'm not sure I understand what you mean, can you clarify? Do you mean avoiding to pass the `$headCommitHash` argument?

Reply:
I mean the performance comparison table in like https://github.com/php/php-src/actions/runs/6054845158?pr=12105 at the end of the page.

For master, it needs to be compared against the latest php-src master commit, for PHP-8.3, it needs to be compared against the latest php-src PHP-8.3 commit.

Reply:
I don't think a change is necessary, we use the `github.event.pull_request.base.sha` variable in push.yml, which is the target branch of whatever PR.